### DISCUSSION TITLE
Help understanding the zero, one, other

### DISCUSSION THREAD

**Discussion by iongion** [i18next/i18next#1677] [Q&A] [answered]
For english, my context count can be `0, 1 or above 1`

```
    containersCount: "There are no containers running",
    containersCount_zero: "There are no containers running",
    containersCount_one: "There is {{count}} container running",
    containersCount_other: "There are {{count}} containers running"
```

But the behavior that I get for `0` is

`t('containersCount', { count: 0 })` => `There are 0 containers running`

I want it to be

`t('containersCount', { count: 0 })` => `There are no containers running`

I am reading the docs and can't wrap my head about it. Using latest

**Comment by jamuhl** [comment, marked as answer]
All based on Intl API which does not use any extra form for zero for languages based on romantic, germanic roots: 

see: https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html

for options to solve this: https://github.com/i18next/i18next/issues/1220

**Reply by iongion** [reply]
Thanks a lot for the detailed explanation, leaving this for the next mortal

Changed to

```
          containersCount: "There are no containers running",
          containersCount_0: "There are no containers running",
          containersCount_one: "There is {{count}} container running",
          containersCount_other: "There are {{count}} containers running"
```

And then the function to `t("containersCount", { count, context: `${count}` })`

Head-spinning!
